poem/hello-world/README.md

```diff
@@ -0,0 +1,11 @@
+# Hello World App with Poem
+This example shows how to build a hello world application using [Poem](https://docs.rs/poem/latest/poem/)
+
+# Run it
+We can start our application using the shuttle CLI.
+Run it locally: `cargo shuttle run`
+Curl it to test it: `curl localhost: 8000/hello`
+
+
+
+
```

Comment:
It's extra newlines like this we should remove, it's correct to have newlines between paragraphs of text, and it's also a good practice to have one newline at the end of every file.